Source/Core/Core/IOS/IOS.cpp

```diff
@@ -361,7 +362,10 @@ void Kernel::AddStaticDevices()
   AddDevice(std::make_unique<Device::USB_KBD>(*this, "/dev/usb/kbd"));
   AddDevice(std::make_unique<Device::SDIOSlot0>(*this, "/dev/sdio/slot0"));
   AddDevice(std::make_unique<Device::Stub>(*this, "/dev/sdio/slot1"));
-  AddDevice(std::make_unique<Device::USB_HIDv4>(*this, "/dev/usb/hid"));
+  if ((ios_title_id & 0xFF) == 59)
```

Comment:
I was more thinking of adding flags to each IOS version details (maybe in MemoryValues? probably would have to be renamed then), so for example:

* IOS57: `Features::NEW_USB_MODULES | Features::SDv2`
* IOS58: `IOS57_FEATURES | Features::EHCI`
* IOS59: `IOS59_FEATURES | Features::WFS`